crates/ruff_linter/src/rules/pyupgrade/rules/super_call_with_parameters.rs

```diff
@@ -42,8 +44,10 @@ use crate::{AlwaysFixableViolation, Edit, Fix};
 ///
 /// ## Fix safety
 ///
-/// This rule's fix is marked as unsafe because removing the arguments from a call
-/// may delete comments that are attached to the arguments.
+/// This rule's fix does not preserve comments.
+///
+/// In [preview], the fix is marked safe if no comments are attached to the arguments.
+/// Else, the fix is marked unsafe.
```

Comment:
I think I kind of preferred the old text for the first sentence/paragraph here. The second paragraph looks good, though.

I think you'll also need a reference link for `[preview]`, at least that's what I saw in other rules with a quick grep.

```suggestion
/// This rule's fix is marked as unsafe because removing the arguments from a call
/// may delete comments that are attached to the arguments.
///
/// In [preview], the fix is marked safe if no comments are present.
///
/// [preview]: https://docs.astral.sh/ruff/preview/
```

I guess we can do without the very last sentence if we bring back the old first bit.